src/test/java/guitests/guihandles/BookCardHandle.java

```diff
@@ -63,4 +64,8 @@ public String getTitle() {
                 .map(Label::getText)
                 .collect(Collectors.toList());
     }
+
+    public boolean equals(Book book) {
+        return getTitle().equals(book.getTitle().title);
```

Comment:
Check that authors, categories, status, priority, and rating matches? Will probably need to add method comment too.